src/test/javascript/spec/component/programming-exercise/programming-exercise-instructor-status.component.spec.ts

```diff
@@ -53,7 +51,7 @@ describe('ProgrammingExerciseInstructorStatusComponent', () => {
                 latestResultSubject = new Subject();
                 subscribeForLatestResultStub.returns(latestResultSubject);
             });
-    }));
+    });
 
     afterEach(() => {
```

Comment:
If I am not mistaken, we change the specific restoring of single stubs to `sinon.restore()` to avoid that - if new stubs are added - these are not restored after the test run.